From akpm@linux-foundation.org  Wed Mar  5 11:40:17 2008
From: Andrew Morton <akpm@linux-foundation.org>
Date: Tue, 04 Mar 2008 15:25:08 -0800
Subject: USB: drivers/usb/storage/sddr55.c: fix uninitialized var warnings
To: greg@kroah.com
Cc: linux-usb@vger.kernel.org, akpm@linux-foundation.org
Message-ID: <200803042325.m24NP87U003202@imap1.linux-foundation.org>


From: Andrew Morton <akpm@linux-foundation.org>

drivers/usb/storage/sddr55.c: In function 'sddr55_transport':
drivers/usb/storage/sddr55.c:526: warning: 'deviceID' may be used uninitialized in this function
drivers/usb/storage/sddr55.c:525: warning: 'manufacturerID' may be used uninitialized in this function

Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/sddr55.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/storage/sddr55.c
+++ b/drivers/usb/storage/sddr55.c
@@ -522,8 +522,8 @@ int sddr55_reset(struct us_data *us) {
 
 static unsigned long sddr55_get_capacity(struct us_data *us) {
 
-	unsigned char manufacturerID;
-	unsigned char deviceID;
+	unsigned char uninitialized_var(manufacturerID);
+	unsigned char uninitialized_var(deviceID);
 	int result;
 	struct sddr55_card_info *info = (struct sddr55_card_info *)us->extra;
 
